spec/features/remove_item_from_cart_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe 'Visitor', type: :feature do
+  feature 'Cart' do
+    create_category
+    create_items
+    it 'can remove item' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.